cli/args/flags.rs

```diff
@@ -846,6 +847,13 @@ fn compile_subcommand<'a>() -> Command<'a> {
   runtime_args(Command::new("compile"), true, false)
     .trailing_var_arg(true)
     .arg(script_arg().required(true))
+    .arg(
+      Arg::new("extra-side-modules")
+        .long("extra-side-modules")
+        .takes_value(true)
+        .multiple_occurrences(true)
```

Comment:
**Bikeshedding:** Since this is multiple occurrence, should it be `--extra-side-module` instead? Plural in name to me suggests a comma separated list.

Also, maybe just `--side-module`? Or something like `--dynamic-module-entry-point`  :D

Reply:
I don't really care about the option name, and as I said in the OP, this could very well be an option in `deno.json` instead, or something else. I'll just let the Deno core team (and/or any contractors 😉) pick the name.

Reply:
Since `deno compile` requires an entrypoint (and therefore, could apply to multiple files within a project), then it's probably best for it to be specified as a deno task. So a flag would be most appropriate. I think dropping the word "extra" for brevity and making it `--side-module` sounds good.